### ISSUE TITLE
Feature Request: Tree View on left

### ISSUE THREAD

**Issue by DeeChael** [M2Team/NanaZip#484]
I want a tree view to see all the directories in the compressed file like Bandizip so that I can clearly now where I am and how is this file structured because sometimes I just use unzip app to check file structure (.docx, .epub etc. they are all structured zip files) and I don't wanna uncompress them.

**Comment by Hexaae** [comment]
Agree. Would greatly enhance the appearance and make UI more natural and intuitive
